grazer/src/main/webapp/WEB-INF/spring/spring-security.xml

```diff
@@ -1,22 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans:beans xmlns="http://www.springframework.org/schema/security"
-	xmlns:beans="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns:context="http://www.springframework.org/schema/context"
-	xsi:schemaLocation="http://www.springframework.org/schema/beans
-	http://www.springframework.org/schema/beans/spring-beans-3.2.xsd
-	http://www.springframework.org/schema/security
-	http://www.springframework.org/schema/security/spring-security-4.1.xsd">
+    xmlns:beans="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:context="http://www.springframework.org/schema/context"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans-3.2.xsd
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+    http://www.springframework.org/schema/security
+    http://www.springframework.org/schema/security/spring-security-4.1.xsd">
 
-	<beans:bean id="encoder"
-		class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder">
-		<beans:constructor-arg name="strength" value="4"></beans:constructor-arg>
-	</beans:bean>
+    <context:component-scan base-package="edu.asu.diging.grazer.authentication" />
 
-	<authentication-manager alias="authenticationManager"
-		erase-credentials="false">
-		<authentication-provider>
-			<user-service id="userDetailsService" properties="classpath:/user.properties" />
-		</authentication-provider>
-	</authentication-manager>
+    <http auto-config="true" use-expressions="true" disable-url-rewriting="true">
+        <form-login login-page="/login" default-target-url="/home"
+            authentication-failure-url="/loginfailed" authentication-success-handler-ref="authSuccessHandler" />
+        <logout logout-success-url="/" invalidate-session="true" />
+    </http>
```

Comment:
since we're using java config for almost everything else for spring security, let's use that for these configurations as well.